package.json

```diff
@@ -3,46 +3,39 @@
   "description": "A grunt plugin to replace url from static files such as img,js,css an put inline in a template",
   "version": "0.1.6",
   "homepage": "https://github.com/dayvson/grunt-static-inline",
-  "author": {
-    "name": "Maxwell Da Silva",
-    "email": "dayvson@gmail.com"
-  },
+  "author": "Maxwell Da Silva <dayvson@gmail.com>",
+  "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/dayvson/grunt-static-inline"
+    "url": "git+https://github.com/dayvson/grunt-static-inline.git"
```

Comment:
this url is wrong it should be:
git://github.com/dayvson/grunt-static-inline.git